src/sonic-yang-models/yang-models/sonic-copp.yang

```diff
@@ -163,6 +163,12 @@ module sonic-copp {
 					}
 					description "reference to CoPP group";
 				}
+
+				leaf always_enabled {
+					mandatory false;
```

Comment:
I don't think you need to explicitly state mandatory false here. Please remove.

Reply:
Done